hermes-2.cxx

```diff
@@ -878,6 +899,10 @@ int Hermes::rhs(BoutReal t) {
     sheath_model = 0;
   }
 
+  if (ramp_j_diamag < 1.0) {
+    ramp_j_diamag = ramp_j_diamag_generator->generate(0, 0, 0, 20 * t/NOUT);
```

Comment:
Here t will be the simulation time, not the output step; NOUT is the number of steps, but t will go to NOUT * TIMESTEP. I'd prefer to just remove NOUT, since any normalisation like this can be done in the input file expression. I suggest here just have `generate(0, 0, 0, t);` and then change the expression in BOUT.inp.